py/client/tests/test_plugin_client.py

```diff
@@ -96,6 +101,52 @@ def test_publish_fetch(self):
 
         plugin_client.close()
 
+    @unittest.skip("need to wait for https://github.com/deephaven/deephaven-core/issues/5996")
```

Comment:
Should we follow this pattern? Or `self.skipTest("need to wait for https://github.com/deephaven/deephaven-core/issues/5996")`? Which is ebtter?

Reply:
the former is for skipping a subtest.

Reply:
Since we are skipping all the subtests, I've changed to use the latter.